.pipelines/load-tests-template.yaml

```diff
@@ -0,0 +1,125 @@
+jobs:
+  - job:
+    displayName: aks-load-test
+    timeoutInMinutes: 360
+    cancelTimeoutInMinutes: 30
+    workspace:
+      clean: all
+    variables:
+    - name: HOME
+      value: $(System.DefaultWorkingDirectory)
+    - name: GOPATH
+      value: $(HOME)/go
+    - name: perf-tests.repo.path
+      value: $(GOPATH)/src/k8s.io/perf-tests
+    - group: csi-secrets-store-load
+    steps:
+      - task: GoTool@0
+        inputs:
+          version: '1.14.1'
+
+      - bash: |
+          git clone -b $(checkout.branch) $(checkout.repo) $(perf-tests.repo.path)
+        displayName: Checkout $(checkout.repo) @ $(checkout.branch)
+
+      - template: templates/build-images.yaml
+
+      - script: |
+          az account set -s=$(LOAD_TEST_SUBSCRIPTION_ID)
+        displayName: "Set subscription for load tests"
+
+      - template: templates/aks-setup.yaml
+
+      - script: |
+          VMSS_NAMES=$(az vmss list -g ${CLUSTER_RESOURCE_GROUP} --query "[].name" -o tsv)
+
+          echo "Assigning /subscriptions/$(LOAD_TEST_SUBSCRIPTION_ID)/resourcegroups/$(USER_MSI_RESOURCE_GROUP)/providers/Microsoft.ManagedIdentity/userAssignedIdentities/$(USER_MSI_NAME) to ${VMSS_NAMES}"
+          # Assign user assigned identity to the VMSS
+          echo "$VMSS_NAMES" | while IFS= read -r VMSS_NAME; do az vmss identity assign -g ${CLUSTER_RESOURCE_GROUP} -n $VMSS_NAME --identities /subscriptions/$(LOAD_TEST_SUBSCRIPTION_ID)/resourcegroups/$(USER_MSI_RESOURCE_GROUP)/providers/Microsoft.ManagedIdentity/userAssignedIdentities/$(USER_MSI_NAME) >> /dev/null; done
+        displayName: "[AKS] Assign User MSI to the VMSS"
```

Comment:
can this be templates? it is in both e2e and load tests

Reply:
Updated!